src/backend/distributed/connection/remote_commands.c

```diff
@@ -191,7 +195,7 @@ ReportConnectionError(MultiConnection *connection, int elevel)
 	int nodePort = connection->port;
 
 	ereport(elevel, (errmsg("connection error: %s:%d", nodeName, nodePort),
-					 errdetail("%s", PQerrorMessage(connection->pgConn))));
+					 errdetail("%s", pchomp(PQerrorMessage(connection->pgConn)))));
```

Comment:
FWIW `PQerrorMessage` is also invoked in `MultiClientSendQuery`, `ReportCopyError`, and `StoreErrorMessage`. I'm not sure which of those go on to `ereport` using the string they find, but it's worth checking.

Reply:
Yeah, that makes sense.


Actually, I changed all of them. Although the error messages have slightly changes for `StoreErrorMessage ()` [citus_tools], I think this is safer.

Are you OK with that?

Reply:
Yeah; that all makes sense.